File: Domain/Entities/Visit.cs
using Domain.Base;

namespace Domain.Entities
{
    public class Visit : IEntityBase
    {
        public Guid Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public string Complaints { get; set; }
        public string Examinations { get; set; }
        public string Diagnosis { get; set; }
        public string Therapy { get; set; }
        public string RequiredAnalysis { get; set; }
        public string Advice { get; set; }
        public string Remarks { get; set; }
        public Patient Patient { get; set; }
        public Guid PatientId { get; set; }
        public Doctor Doctor { get; set; }
        public Guid DoctorId { get; set; }
       
    }
}

File: client/src/app/models/Visit.ts
import Doctor from "./Doctor";
import Patient from "./Patient";

export default interface Vist {
    id: string;
    createdAt: Date;
    updatedAt: Date;
    complaints: string;
    examinations: string;
    diagnosis: string;
    therapy: string;
    requiredAnalysis: string;
    advice: string;
    remarks: string;
    doctorId: string;
    doctor: Doctor;
    patientId: string;
    patient: Patient;

}
